dev/merge_spark_pr.py

```diff
@@ -246,6 +246,9 @@ def resolve_jira_issue(merge_branches, comment, default_jira_id=""):
     jira_id = input("Enter a JIRA id [%s]: " % default_jira_id)
     if jira_id == "":
         jira_id = default_jira_id
+        if jira_id == "":
```

Comment:
We already have the same condition in line 247.  Do we need this `if` statement?